public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuWrapper.tsx

```diff
@@ -11,12 +11,12 @@ interface Props {
   onClose: () => void;
 }
 
-export const PanelHeaderMenuWrapper: FC<Props> = ({ panel, dashboard }) => {
+export function PanelHeaderMenuWrapper({ panel, dashboard }: Props) {
   return (
     <PanelHeaderMenuProvider panel={panel} dashboard={dashboard}>
       {({ items }) => {
         return <PanelHeaderMenu items={items} />;
```

Comment:
My hope / plan was to not use this component as it is using some very old menu styles / css classes, and is not using our Menu component from grafana/ui 

But we can do that switch in another PR